### PR TITLE
[tests-only][full-ci]Remove function present in middleware from webDavHelper

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
This PR removes functions present in middleware that are no longer needed in web

## Related Issue

- part of: https://github.com/owncloud/web/issues/6111

## How Has This Been Tested?

- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by SwikritiT** [comment]
The CI is green. We can also remove the `download` function after this step is refactored by PR https://github.com/owncloud/web/pull/6188, we can wait until that PR is merged or we can merge this PR and remove the `download` function in the same PR that refactored the steps.
https://github.com/owncloud/web/blob/af5b48f9c44934655744dac9947b7773d14f4fee/tests/acceptance/stepDefinitions/generalContext.js#L135-L145

**Review by phil-davis — APPROVED** [review]
deleted code is good

**Comment by phil-davis** [comment]
> We can also remove the download function after this step is refactored by PR #6188 

I will merge this. @kiranparajuli589 can sort out the remaining code deletion in his PR.

**Pull request merged**
